src/main/webapp/js/user-page-loader.js

```diff
@@ -95,4 +96,21 @@ function buildUI() {
   setPageTitle();
   showMessageFormIfViewingSelf();
   fetchMessages();
-}
\ No newline at end of file
+  fetchAboutMe();
+}
+
+/** Fetches about me data */
+function fetchAboutMe(){
+    const url = '/about?user=' + parameterUsername;
```

Comment:
Where does "parameterUsername" come from?

Here's what I'd suggest....

Organize the application using the following URLs:

      /profiles/me[?mode=VIEW|EDIT]
      /profiles/USER_NAME

In most of your code, I would make the parameters explicit (like `fetchProfileData(username)`, where the username is an explicit function parameter). Then, separately, you can have a function that explicitly retrrieves the parameter from the URL as in `extractUsernameFromProfileUrl(url)`, allowing for a fairly untuitive combination like:

```
    var url = window.location.href;
    var username = extractUsernameFromProfileUrl(url);
    fetchProfileData(username)
         .then(handleProfileDataReceived)
         .catch(handleProfileDataFetchingFailed);
```

Reply:
parameterUsername comes from line 19 as a FYI,
But I do like what Michael is proposing, it makes it where you just have to look around the code to understand what is going on instead of reading the entire file.